src/infiniteGrid.js

```diff
@@ -754,9 +767,7 @@ eg.module("infiniteGrid", ["jQuery", eg, window, document, "Outlayer"], function
 			var y = this.core.updateCols();	// for prepend
 			while (item = this.core.items[i++]) {
 				item.position.y -= y;
-				applyDom && item.css({
-					"top": item.position.y + "px"
-				});
+				applyDom && $.style(item.element, "top", item.position.y + "px");
```

Comment:
no. but, $.style is a bit fast. 
$.style, $.css have been used in egjs and jQuery internal.